scene/add-3d-tiles-layer/README.metadata.json

```
@@ -0,0 +1,29 @@
+{
+    "category": "Scene",
+    "description": "Add a layer to visualize 3D tiles data that conforms to the OGC 3D Tiles specification.",
+    "ignore": false,
+    "images": [
+        "Add3dTilesLayer.png"
+    ],
+    "keywords": [
+        "3d tiles",
+        "layers",
+        "OGC",
+        "OGC API",
+        "scene",
+        "service",
+        "Ogc3DTilesLayer",
+        "SceneView"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/add-3d-tiles-layer.htm"
+    ],
```

Because this is a new sample it doesn't need a redirect so can just be:
 
```suggestion
    "redirect_from": "",
```

---

Thank you for letting me know, I shall remember for future reference!